lib/Infer/AliveDriver.cpp

```diff
@@ -70,7 +70,7 @@ class FunctionBuilder {
   template <typename T>
   void assume(T &&V) {
     auto AI = std::make_unique<IR::Assume>(*std::move(V));
-    F.getBB("").addIntr(std::move(AI));
+    F.getBB("").addInstr(std::move(AI));
```

Comment:
you probably should rebase your branch since you made these changes in a different PR, right?